src/Lesson1/HomeWorkApp.java

```diff
@@ -0,0 +1,37 @@
+package Lesson1;
+
+public class HomeWorkApp {
+    public static void main(String[] args) {
+        printThreeWords();
+        checkSumSign();
+        printColor();
+        compareNumbers();
+    }
+    private static void printThreeWords(){
+        System.out.println("Orange\nBanana\nApple");
+    }
+    private static void checkSumSign(){
+        int a = 3;
+        int b = 5;
+        if ((a+b) >= 0){
+            System.out.println("Сумма положительная");
+        } else System.out.println("Сумма отрицательная");
+    }
+    private static void printColor(){
+        int value = 45;
+        if (value <= 0){
+            System.out.println("Красный");
+        } else if (value > 0 && value<= 100){
```

Comment:
```suggestion
        } else if (value > 0 && value <= 100) {
```
старайтесь использовать пробел для отделения конструкций, это повышает читаемость кода
Можете использовать ctrl+alt+L для автоформатирования